controller/src/test/java/io/pravega/controller/server/eventProcessor/SerializedRequestHandlerTest.java

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
```

Comment:
Please do not change the formatting of the license header.

Reply:
done